lib/events-reader.js

```diff
@@ -189,10 +189,10 @@ module.exports = function (harvesterApp) {
                         });
 
                 } else {
-                    that.reschedule(1000);
+                    that.reschedule(0);
                 }
             } else {
-                that.reschedule(1000);
+                that.reschedule(0);
```

Comment:
0ms should get you to nextTick, and should not block anything, but yes, the server will loop furiously. But my goal is that the queue is almost always empty (which just means events-reader gets to work fast and the change-handlers work fast), but if that's the case, we can expect most of those events to take an additional 250ms (avg wait inflicted by 500ms delay).

I think the oplog is consumed via tailable cursor, so it's not costing us anything extra on the mongo server to read this often.